android/manifest.xsl

```diff
@@ -8,19 +8,19 @@
 	<xsl:template match="comment()" />
 
 	<xsl:template match="meta-data[@android:name='FACEBOOK_APP_ID']">
-		<meta-data android:name="FACEBOOK_APP_ID" android:value="\ {$facebook.facebookAppID}"/>
+		<meta-data android:name="FACEBOOK_APP_ID" android:value="{$facebook.facebookAppID}"/>
```

Comment:
does this work?
Previously the \ was added so that numbers are also considered as strings.

Reply:
Sometimes additional slash makes app id worng. We should test facebook plugin well and completely.